presto-product-tests/conf/tempto/tempto-configuration.yaml

```diff
@@ -3,3 +3,23 @@ databases:
     host: hadoop-master
   presto:
     host: presto-master
+
+  prestoConfigUserAlice:
+    host: localhost
+    server_address: ${databases.presto.host}:8080
+    cli_server_address: ${databases.presto.host}:8080
+    jdbc_driver_class: com.facebook.presto.jdbc.PrestoDriver
+    jdbc_url: jdbc:presto://${databases.presto.server_address}/hive/${databases.hive.schema}
+    jdbc_user: alice
+    jdbc_password: na
+    jdbc_pooling: false
+
+  prestoConfigUserBob:
+    host: localhost
+    server_address: ${databases.presto.host}:8080
+    cli_server_address: ${databases.presto.host}:8080
+    jdbc_driver_class: com.facebook.presto.jdbc.PrestoDriver
+    jdbc_url: jdbc:presto://${databases.presto.server_address}/hive/${databases.hive.schema}
+    jdbc_user: bob
+    jdbc_password: na
+    jdbc_pooling: false
```

Comment:
I don't completely understand the use cases when each of them are used. `run_on_docker.sh` script uses `presto-product-tests/conf/tempto/tempto-configuration.yaml`, right? And when is the `presto-product-tests/src/main/resources/tempto-configuration.yaml` used?